Cesium3DTiles/src/RasterOverlayTileProvider.cpp

```diff
@@ -614,7 +626,8 @@ void RasterOverlayTileProvider::doLoad(
   // Don't let this tile be destroyed while it's loading.
   tile.setState(RasterOverlayTile::LoadState::Loading);
 
-  this->beginTileLoad(tile, isThrottledLoad);
+  [[maybe_unused]] int64_t loadID = this->beginTileLoad(tile, isThrottledLoad);
+  CESIUM_TRACE_ASYNC_ENLIST(loadID);
```

Comment:
I'm curious what is the purpose of this macro? It looks like it's not needed since we don't use CESIUM_TRACE_BEGIN and CESIUM_TRACE_END in this class

Reply:
Does the documentation on it help at all? It says "this scope in this thread is part of an async process with this ID". That async process ID is then used for any non-ID TRACE calls, as well inherited by any work dispatched through the AsyncSystem.